server/routes/authRouter.js

```diff
@@ -96,48 +184,32 @@ authRouter.post('/login', passport.authenticate('local', {
 
 
 authRouter.route('/success').get((req, res) => {
-  console.log('success');
-  res.status(200).send(true);
+  console.log('success:', req.session);
+  if (req.session.passport.user.username) {
+      res.status(200).send(true);
+  } else {
+    res.status(200).send(false);
+  }
 });
 
 authRouter.get('/isLoggedIn', (req, res) => {
   let isAuth = req.isAuthenticated();
-  // console.log('isLoggedIn auth:', isAuth,  req.session);
-  res.status(200).send(isAuth)
-});
-
-// optimize later with middleware that verifies this and /isloggedin
-authRouter.get('/userEmail', (req, res) => {
-  let isAuth = req.isAuthenticated();
-  // console.log('user:', req.user);
-  // res.locals.currentUser = req.user;
+  console.log('isLoggedIn auth:', isAuth,  req.session);
   if (isAuth) {
-    res.status(200).send(req.user);
+    res.status(200).send({loggedIn: isAuth, info: req.session.passport.user.username})
   } else {
-    res.status(200).send(null);
+    res.send(isAuth);
   }
-
 });
 
+// optimize later with middleware that verifies this and /isloggedin
+
 authRouter.route('/failure').get((req, res) => {
```

Comment:
i think it's safe to delete this and similar others in authrouter.  was using it for testing.